app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,57 @@
+class OrdersController < ApplicationController
+    before_action :order_page_access, only: [:index]
+
+  def index
+    @merchant = session[:user_id]
+    @orders = Order.includes(:order_items).where(order_items: { user_id: @merchant } )
+    # @order_item = OrderItem.find(params[:id])
+
+    if params[:status] == "paid"
+      @orders = @orders.where("status = ?", "paid")
+    elsif params[:status] == "complete"
+      @orders = @orders.where("status = ?", "Complete")
+    end
+    render :index
+  end
+
+
+  def edit
+    @order = Order.find(session[:order_id])
```

Comment:
Lots of duplication in this `find` method call so i'd add a `before_action` for each of these controller actions